webapp/src/components/AssetCard/AssetCard.tsx

```diff
@@ -123,8 +228,17 @@ const AssetCard = (props: Props) => {
       ) : null}
       <Card.Content>
         <Card.Header>
-          <div className="title">{title}</div>
-          {price ? (
+          <div className="title">
+            {title}
+            {isCatalogItem(asset) && (
+              <LinkedProfile
+                address={asset.contractAddress}
+                textOnly={true}
+                className="Creator"
```

Comment:
```suggestion
                className="creator"
```
should we use the Camelcase for classes? I think we're using other convention